templates/user/settings/repos.tmpl

```diff
@@ -27,7 +27,9 @@
 											<span class="icon">{{svg "octicon-repo"}}</span>
 										{{end}}
 										<a class="name" href="{{$repo.Link}}">{{$repo.OwnerName}}/{{$repo.Name}}</a>
-										<span>{{FileSize $repo.Size}}</span>
+										<span  class="ui{{if not (eq $repo.LFSSize 0)}} tooltip{{end}}"{{if not (eq $repo.LFSSize 0)}} data-content="{{$repo.SizeDetailsString}}"{{end}}>
```

Comment:
```suggestion
										<span  class="ui{{if not (eq $repo.Size 0)}} tooltip{{end}}"{{if not (eq $repo.Size 0)}} data-content="{{$repo.SizeDetailsString}}"{{end}}>
```

Reply:
same as before if no lfs no need to show the tooltip so we can leave like it was